### PR TITLE
Add kbdlight package and setuid wrapper

### PR DESCRIPTION
**Pull request by womfoo**
console app to control the keyboard backlight

### CODE REVIEW

**Comment by womfoo** [comment]
Tested OK on a MacBookPro11,3

File: nixos/modules/programs/kbdlight.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+with lib;
+
+let
+  cfg = config.programs.kbdlight;
+
+in
+{
+  options = {
+    programs.kbdlight = {
+      enable = mkOption {
```
Comment (jagajaga) [inline nixos/modules/programs/kbdlight.nix]: There is `mkEnableOption`.

**Comment by lethalman** [comment]
Travis passed and the module looks good to me. Thanks and welcome.

**Comment by womfoo** [comment]
Thank you! @lethalman @jagajaga

**Pull request merged**